src/main/java/org/jvnet/hudson/plugins/seleniumhtmlreport/SeleniumHtmlReportAction.java

```diff
@@ -17,11 +15,11 @@
  */
 public class SeleniumHtmlReportAction implements Action, Serializable {
 
-    public final AbstractBuild<?, ?> build;
+    public final Run<?, ?> build;
```

Comment:
Make this `transient` and implement `RunAction2` to set it rather taking a build in the constructor.